.github/workflows/ci.yml

```diff
@@ -33,6 +33,8 @@ jobs:
         env:
           VALIDATE_ALL_CODEBASE: false
           VALIDATE_LATEX: false # template.tex is not tex, but a template with Python code
+          VALIDATE_PYTHON_ISORT: false # because black and isort are inconsistent
+          VALIDATE_PYTHON_BLACK: false # because local black and black are different
```

Comment:
They probably need to be configured: https://pycqa.github.io/isort/docs/configuration/black_compatibility/